src/components/LandingPage/download-data.ts

```diff
@@ -0,0 +1,43 @@
+export interface DownloadInfo {
+    date: string;
+    title: string;
+    link: string;
+}
+
+export default [
+    {
+        date: "May/26/2021",
+        title: "The hiPSC single-cell image dataset",
+        link: "https://open.quiltdata.com/b/allencell/packages/aics/hipsc_single_cell_image_dataset/tree/latest/metadata.csv",
+    },
+    {
+        date: "Jul/21/2022",
+        title: "The hiPSC single i1 cell image dataset",
+        link: "https://open.quiltdata.com/b/allencell/packages/aics/hipsc_single_i1_cell_image_dataset/tree/latest/metadata.csv",
+    },
+    {
+        date: "Jul/20/2022",
+        title: "The hiPSC single M1 cell image dataset",
+        link: "https://open.quiltdata.com/b/allencell/packages/aics/hipsc_single_m1_cell_image_dataset/tree/latest/metadata.csv",
+    },
+    {
+        date: "Jul/21/2022",
+        title: "The hiPSC single i2 cell image dataset",
+        link: "https://open.quiltdata.com/b/allencell/packages/aics/hipsc_single_i2_cell_image_dataset/tree/latest/metadata.csv",
+    },
+    {
+        date: "Jul/20/2022",
+        title: "The hiPSC single M2 cell image dataset",
+        link: "https://open.quiltdata.com/b/allencell/packages/aics/hipsc_single_m2_cell_image_dataset/tree/latest/metadata.csv",
+    },
+    {
+        date: "Jul/19/2022",
+        title: "The hiPSC single edge cell image dataset",
+        link: "https://open.quiltdata.com/b/allencell/packages/aics/hipsc_single_edge_cell_image_dataset/tree/latest/metadata.csv",
+    },
+    {
+        date: "Sep/28/2022",
+        title: "The hiPSC single non-edge cell image dataset",
+        link: "https://open.quiltdata.com/b/allencell/packages/aics/hipsc_single_nonedge_cell_image_dataset/tree/latest/metadata.csv",
+    },
+] as DownloadInfo[];
```

Comment:
Why not in cell-feature-data?  Is there not a 1-1 correspondence with the datasets in cell-feature-data?

Reply:
I think that would be ideal, but I wasn't sure when that dataset was going to be merged and I thought this would be faster. I also wasn't sure which one is which: https://github.com/allen-cell-animated/cell-feature-data/pull/39/files#diff-0cb6b75f6f6c7fa3d607d9ecd4a83afbb699c0525454ff3cee9a48a49d921bc0 has only 4 new datasets and there are 7 here.